lib/src/report/multi_platform.dart

```diff
@@ -178,3 +184,49 @@ Future<Subsection> _createWasmSubsection(PackageContext context) async {
     );
   }
 }
+
+/// Create a subsection for ios and macos plugins, to highlight supported
+/// for swift package manager (or lack there of).
+Future<Subsection?> _createSwiftPackageManagerSubSection(
+    PackageContext context) async {
+  final tr = await context.staticAnalysis;
+  final description = 'Swift Package Manager support';
+
+  if (tr.tags.contains(PanaTags.isSwiftPmPlugin)) {
+    return Subsection(
+      description,
+      [
+        RawParagraph(
+            'This iOS or MacOs plugin supports the Swift Package Manager. '
+            'It will be rewarded additional points in a future version of the scoring model.'),
+        RawParagraph(
+            'See https://docs.flutter.dev/packages-and-plugins/swift-package-manager/for-plugin-authors for details.'),
```

Comment:
Consider linking to `https://docs.flutter.dev/to/spm` instead. This redirects to the docs we have on SwiftPM. If we ever move the docs location, we'll update that redirect so that we don't break tooling that links to it.

Reply:
Done